### PR TITLE
fix: update deps

### PR DESCRIPTION
**Pull request by spaenleh**

### CODE REVIEW

File: cypress/e2e/appAnalytics.cy.ts [hunk 1]
```diff
@@ -13,24 +13,17 @@ import MOCK_ITEMS, {
...
+  cy.get(`#${buildSidebarListItemId(APP_ITEM)}`).click();
 };
 
 const checkContainAppAnalytics = () => {
   cy.get(`#${buildSidebarListItemId(APP_ITEM)}`).should('exist');
   cy.get(`#${APPS_ID}`).should('exist');
 };
 
-describe('Check An App Item has an app analytics ', () => {
+describe.only('Check An App Item has an app analytics ', () => {
```
Comment (ReidyT) [inline cypress/e2e/appAnalytics.cy.ts:26]:
```suggestion
describe('Check An App Item has an app analytics ', () => {
```

File: package.json [hunk 2]
```diff
@@ -18,35 +18,35 @@
     "@emotion/cache": "11.13.1",
     "@emotion/react": "11.13.3",
     "@emotion/styled": "11.13.0",
-    "@graasp/query-client": "3.22.2",
-    "@graasp/sdk": "4.26.0",
-    "@graasp/translations": "1.35.1",
-    "@graasp/ui": "4.26.0",
+    "@graasp/query-client": "4.0.1",
+    "@graasp/sdk": "github:graasp/graasp-sdk#add-member-util",
```
Comment (ReidyT) [inline package.json:22]: reminder ^^

File: src/components/App.tsx [hunk 3]
```diff
@@ -31,78 +35,67 @@ import UsersAnalyticPage from './pages/Item/UsersAnalyticPage';
...
   const { i18n } = useTranslation();
-  const { pathname } = useLocation();
 
   useEffect(() => {
-    if (currentMember?.extra?.lang !== i18n.language) {
-      i18n.changeLanguage(currentMember?.extra?.lang ?? 'en');
+    const lang =
+      currentAccount?.type === AccountType.Individual
+        ? currentAccount?.extra?.lang || DEFAULT_LANG
+        : DEFAULT_LANG;
```
Comment (ReidyT) [inline src/components/App.tsx:45]: Do we have a function for that now? I think I reviewed a util function?

File: src/components/context/DataProvider.tsx [hunk 4]
```diff
@@ -257,9 +256,7 @@ const DataProvider = ({ children }: Props): JSX.Element => {
     ],
   );
 
-  return (
-    <DataContext.Provider value={value}> {children} </DataContext.Provider>
-  );
+  return <DataContext.Provider value={value}> {children}</DataContext.Provider>;
```
Comment (ReidyT) [inline src/components/context/DataProvider.tsx:259]:
```suggestion
  return <DataContext.Provider value={value}> {children} </DataContext.Provider>;
```

**Review by ReidyT — APPROVED** [review]
Thanks for this PR, LGTM 🦄

File: src/components/context/DataProvider.tsx (hunk 4 above)
Comment (spaenleh) [reply src/components/context/DataProvider.tsx:259]: I don't know why it is formatted like that ...

**Pull request merged**